Source/WebCore/Modules/fetch/FetchHeaders.cpp

```diff
@@ -57,7 +57,7 @@ static ExceptionOr<bool> canWriteHeader(const String& name, const String& value,
     return true;
 }
 
-static ExceptionOr<void> appendToHeaderMap(const String& name, const String& value, HTTPHeaderMap& headers, FetchHeaders::Guard guard)
+static ExceptionOr<void> appendToHeaderMap(const String& name, const String& value, HTTPHeaderMap& headers, Vector<String>& setCookieValues, FetchHeaders::Guard guard)
```

Comment:
It might be handy to introduce a struct InternalHeaders {
HTTPHeaderMap;
Vector<String>&;
};

Reply:
Is the idea to use this struct on `FetchHeaders::internalHeaders()` and `setInternalHeaders()` as well? As far as I can tell, those methods are only used for fetch requests and responses, and shouldn't need to get or set the `Set-Cookie` values.

Reply:
Yeah, but this seems good enough the way you did it.